CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## [7.2.0] (IN PROGRESS)
 
```

Comment:
Thank you for adding it. Just one note is that the new changes should be added to the end of the list in progress. So the line for `UIEH-1269` should be after the line for `UIEH-1265`.